bigquery_etl/cli/backfill.py

```diff
@@ -122,6 +122,13 @@ def backfill(ctx):
     "--shredder_mitigation/--no_shredder_mitigation",
     help="Wether to run a backfill using an auto-generated query that mitigates shredder effect.",
 )
+@click.option(
+    "--override-retention-range-limit",
+    required=True,
+    type=bool,
+    help="True to allow running a backfill outside the retention policy limit.",
+    default=False,
+)
```

Comment:
I guess this could also be a flag. `is_flag=True`

Then just `--override-retention-range-limit` would suffice to override the range.

Reply:
Just pushed this - thanks for the suggestion!